Extension/src/LanguageServer/client.ts

```diff
@@ -868,6 +877,87 @@ export class DefaultClient implements Client {
         return workspaceFolder ? workspaceFolder.name : "untitled";
     }
 
+    public updateClientConfigurations(): void {
+        clients.forEach(client => {
+            if (client instanceof DefaultClient) {
+                const defaultClient: DefaultClient = <DefaultClient>client;
+                defaultClient.configuration.CompilerDefaults = compilerDefaults;
+                defaultClient.configuration.handleConfigurationChange();
+            }
+        });
+    }
+
+    public async handleCompilerQuickPick(): Promise<void> {
+        const compilerPath: PersistentState<string[]> = new PersistentState<string[]>("CPP.trustedCompilerPaths", []);
+        const settings: OtherSettings = new OtherSettings();
+        const paths: string[] = compilerDefaults.knownCompilers.map(function (a: configs.KnownCompiler): string { return a.path; });
+        paths.push(localize("selectAnotherCompiler.string", "Select another compiler on my machine"));
+        paths.push(localize("installCompiler.string", "Help me install a compiler"));
+        paths.push(localize("noConfig.string", "Do not configure a compiler (not recommended)"));
+        const index: number = await ui.showSelectCompiler(paths);
+        if (index === paths.length - 1) {
+            paths[index] = "";
+        } else if (index === paths.length - 2) {
+            switch (os.platform()) {
+                case 'win32':
+                    vscode.commands.executeCommand('vscode.open', "https://go.microsoft.com/fwlink/?linkid=2217614");
```

Comment:
This opens the "Using GCC with MinGW" docs -- can/should we create a new doc page that is specific to installing compilers on Windows, which would include info on how to install the cl.exe compiler, the mingw compiler, the LLVM clang compilers, or maybe whatever other compilers we see high enough usage data for?

Reply:
This is a question for @AlexandraKemperMS

Reply:
Also, I don't think it's necessary for V1, but it'd be a nice follow up improvement in regards to docs and the compiler acquisition story.